packages/ruby-2.7/packaging

```diff
@@ -15,7 +15,7 @@ if [ $(uname -s) == "Darwin" ]; then
 fi
 
 # Use Clang if available; the resulting Ruby is faster
-if [ -x /usr/bin/clang ]; then
+if command -v clang &> /dev/null; then
   export CXX=/usr/bin/clang++ CC=/usr/bin/clang
```

Comment:
Whenever we find `clang` anywhere else than `/usr/bin`, why would we infer there is one in `/usr/bin`?

Seems the env var setup is not aligned with the actual location where clang is found.

Reply:
Thank you! Fix done!